lib/Reader/readerir.cpp

```diff
@@ -5479,6 +5479,22 @@ bool GenIR::abs(IRNode *Argument, IRNode **Result) {
   return false;
 }
 
+bool GenIR::sqrt(IRNode *Argument, IRNode **Result) {
+  Type *Ty = Argument->getType();
+
+  if (Ty->isFloatingPointTy()) {
+    Type *Types[] = {Ty};
+    Value *FSqrt = Intrinsic::getDeclaration(JitContext->CurrentModule,
+                                             Intrinsic::sqrt, Types);
```

Comment:
Ok.  Thanks for verifying.

I assume that if we do hit issues with other platforms we'll want to use this intrinsic and add the right lowering under our environment anyway.